package/doc/sphinx/source/conf.py

```diff
@@ -46,7 +47,7 @@
 #       are served from https://www.mdanalysis.org/docs/ and the sitemap.xml
 #       is manually fixed when deploying the release docs with the
 #       maintainer/deploy_master_docs.sh script
-site_url = "https://www.mdanalysis.org/mdanalysis/"
+site_url = "https://docs.mdanalysis.org/"
```

Comment:
Add comment regarding our DNS set up (requires a CNAME etc). The previous "https://www.mdanalysis.org/mdanalysis/" would work automatically with the standard GitHub setup where any gh-pages repo would appear under our top level. With the new set up we need correct DNS.